api/resolvers/notifications.js

```diff
@@ -320,9 +320,31 @@ export default {
           FROM "Streak"
           WHERE "userId" = $1
           AND updated_at < $2
+          AND type = 'COWBOY_HAT'
           ORDER BY "sortTime" DESC
           LIMIT ${LIMIT})`
         )
+        for (const type of ['HORSE', 'GUN']) {
+          queries.push(
+            `(SELECT (id::text || '-${type}-FOUND')::text AS id, "startedAt" AS "sortTime", 0 as "earnedSats", 'Streak' AS type
```

Comment:
Since the same row is used to detect found and lost, I am encoding in the id if it was found or lost.

I then use that information when resolving `days` to return a number > 0 (lost) or not (found).

Reply:
Because the same row is used to generate two notifications, so their id needs to be different:

> Since the same row is used to detect found and lost, I am encoding in the id if it was found or lost.

Reply:
Refactored this in c15c317a. Thanks for the feedback, looks much better now!